sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -66,6 +66,14 @@
 #define STABILIZATION_INDI_FILT_CUTOFF_R 20.0
 #endif
 
+#ifndef STABILIZATION_INDI_ALLOCATION_PSEUDO_INVERSE
+#define STABILIZATION_INDI_ALLOCATION_PSEUDO_INVERSE TRUE
```

Comment:
Currently it is not defined by default, making the #if statements false, so that WLS is used. Do we want to change the default behavior?

Reply:
If NOT defined, then pseudoinverse will be true and used, and things will be called just like before. Setting the ```STABILIZATION_INDI_ALLOCATION_PSEUDO_INVERSE``` to ```FALSE``` is needed to solve non-invertible problems.

Reply:
I am quite sure that if you have an ```#if``` statement, it defaults to false if the macro is not defined. Now you're explicitly defining STABILIZATION_INDI_ALLOCATION_PSEUDO_INVERSE to TRUE if it is not defined

Reply:
OK, I did not see this define was already there.